app/controllers/topics_controller.rb

```diff
@@ -472,7 +472,7 @@ def timer
     }
 
     options.merge!(category_id: params[:category_id]) if !params[:category_id].blank?
-    options.merge!(duration: params[:duration].to_i) if params[:duration].present?
+    options.merge!(duration_minutes: params[:duration_minutes].to_i) if params[:duration_minutes].present?
```

Comment:
This is technically a breaking change if someone is relying on the old API. We might not care about that since it's an advanced thing to do programatically but it's worth calling out.

Reply:
Good point, will add this back because I had to keep backwards compat anyway